internal/pkg/env/map_test.go

```diff
@@ -53,7 +53,7 @@ func TestEnvMap(t *testing.T) {
 	m.Set(`X`, `Y`)
 	str, err = m.ToString()
 	assert.NoError(t, err)
-	assert.Equal(t, "A=\"123\"\nX=\"Y\"", str)
+	assert.Equal(t, "A=123\nX=\"Y\"", str)
```

Comment:
V godotenv 1.4 se exportují celý čísla bez uvozovek (https://github.com/joho/godotenv/releases). Testy jsou ok, tak snad to nic nerozbije. 🤔